store/models/job_spec.go

```diff
@@ -56,6 +56,15 @@ func NewJob() JobSpec {
 	}
 }
 
+func NewJobFromRequest(jsr JobSpecRequest) JobSpec {
```

Comment:
We will need to document this method. Go convention is that the first word is the method name. Caught by Code Climate but lost in the noise of the other errors.

Reply:
Thanks for catching this. Added a docstring.